### DISCUSSION TITLE
MFA Extension for NPS (RADIUS)

### DISCUSSION THREAD

**Discussion by derSchweiger** [neos-sdi/adfsmfa#302] [Q&A] [answered]
Hello,

just a quick question about the plans for implementing a similar MFA extension for Windows Network Policy Servers to support RADIUS as an additional protocol? Since this ADFS MFA extension is working perfectly and integrates seamlessly into existing infrastructures, it could be a perfect fit to support RADIUS too.

**Comment by redhook62** [comment, marked as answer]
Hi,

I admit that I don't have much time at the moment.
In the future version 4.0 which will above all be a refactoring of 3.1 (with some nice technical new features all the same...), it is planned to support 1st factor authentication plugins (aka login/pwd) such as client certificates and I will think about Radius (as a first or second factor).
